llvm/lib/Target/AIE/AIEMachineScheduler.cpp

```diff
@@ -544,6 +544,13 @@ void AIEPostRASchedStrategy::enterMBB(MachineBasicBlock *MBB) {
 void AIEPostRASchedStrategy::commitBlockSchedule(MachineBasicBlock *BB) {
   auto &BS = InterBlock.getBlockState(BB);
 
+  // TODO: Update assert when the fixed instructions become part of the
+  // scheduling region.
+  assert(BS.getRegions().empty() ||
+         0 == BS.getTop().getTopFixedBundles().size());
```

Comment:
There's no empty() for an ArrayRef?

Reply:
Hmm, there is. I guess I wanted to leave the number `0` so I can easily replace it with e.g. `BS.BottomInsert.size()` in a future commit.